tests/regression/runner.py

```diff
@@ -334,7 +409,7 @@ def test_send_transfer(self):
         logging.basicConfig(level=logging.DEBUG)
     else:
         logging.basicConfig(level=logging.INFO)
-    unittest.main(argv=['first-arg-is-ignored'], exit=False)
+    unittest.main(argv=['first-arg-is-ignored'], exit=True)
```

Comment:
What's the point to `sys.exit()` here?

Reply:
The exit can notify CI that there are some errors in python script